README.md

```diff
@@ -287,6 +289,45 @@ timeouts(
 )
 ```
 
+### `throttle_delays(...)`
+
+Configures the speed of virtual keyboard keystroke output to deal with issues that occur in various situations with the timing of modifier key presses and releases being misinterpreted. 
+
+- `key_pre_delay_ms` - The number of milliseconds to delay the press-release keystroke of the "normal" key after pressing modifier keys. 
+- `key_post_delay_ms` - The number of milliseconds to delay the next key event (modifier release) after the "normal" key press-release event.
```

Comment:
I any of your testing did you try putting the delay in the middle, ie holding down the key for a bit...

- delay
- key down
- delay
- key up

Reply:
> did you try putting the delay in the middle

Tried that a number of times. Never seemed to have a benefit. And if you delay the release long enough doesn’t it start repeating? 

Because this is an asynchronicity problem, the secret seems to be just getting enough of a delay to get something to recognize that you pressed the relevant modifier before the normal key, not after, and released the mod after releasing the normal key, not before. I think that’s why a longer post delay is a bit more effective than a long pre delay. But if post is by itself with no pre it has to be longer (for about the same total delay) and that didn’t seem to create as reliable of an “aperture” timing buffer window around the normal keystroke.